docs/source/cli-docs.md

```diff
@@ -0,0 +1,162 @@
+# Kedro Viz CLI reference
+
+The Kedro Viz CLI provides commands to visualise Kedro pipelines, deploy them to cloud platforms, and export the visualisation data. Below is a detailed description of the available commands and options.
+
+## Commands
+
+### `kedro viz`
+
+Launches a local Kedro Viz instance to visualise a Kedro pipeline.
+
+**Usage:**
+
+```bash
+kedro viz [OPTIONS]
+```
+
+**Description:**
+
+This command launches the Kedro Viz server to visualise a Kedro pipeline. It is functionally the same as `kedro viz run`. If no sub-command is provided, `run` is used by default.
+
+**Options:**
+
+This command accepts all the options that are available in the `kedro viz`, `kedro viz run` command. See the `kedro viz run` section for a complete list of options.
+
+### `kedro viz run`
+
+Launches a local Kedro Viz instance to visualise a Kedro pipeline.
+
+**Usage:**
+
+```bash
+kedro viz run [OPTIONS]
+```
+
+**Options:**
+
+- `--host <host>`
+  - Host that Kedro Viz will listen to. Defaults to `localhost`.
+  
+- `--port <port>`
+  - TCP port that Kedro Viz will listen to. Defaults to `4141`.
+
+- `--browser / --no-browser`
+  - Whether to open the Kedro Viz interface in the default browser. The browser will open if the host is `localhost`. Defaults to `True`.
```

Comment:
Do we need this here ? - The browser will open if the host is `localhost`

Reply:
Yes, i got it from the docs. Maybe we can reword it in cli, and then change it here.